content/applications/productivity/iot.rst

```diff
@@ -12,3 +12,14 @@ Internet of Things (IoT)
 
    iot/config
    iot/devices
+
+.. seealso::
```

Comment:
Think a **note** block is better here [to convey additional information like this](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#note).

**seealso** is mostly used to link references and other documentation, so doesn't *quite* fit the use case here I don't think.